parse/src/main/java/com/parse/EncryptedFileObjectStore.java

```diff
@@ -0,0 +1,94 @@
+package com.parse;
+
+import android.content.Context;
+
+import androidx.security.crypto.EncryptedFile;
+import androidx.security.crypto.MasterKey;
+
+import com.parse.boltsinternal.Task;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.File;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.concurrent.Callable;
+
+public class EncryptedFileObjectStore<T extends ParseObject> implements ParseObjectStore<T> {
```

Comment:
Can you add Javadoc on the public classes and their public methods you have introduced?